docs/guide/list-of-differences.md

```diff
@@ -20,20 +20,21 @@ See a full list of differences between HyperFormula, Microsoft Excel, and Google
+|----------------------------------------------------|---------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------|
+| Dependency collection                              | A1:=IF(FALSE(), A1, 0)<br><br>ISREF(A1)                                   | Dependencies are collected during the parsing phase, which finds cycles that wouldn't appear in the evaluation.<br><br>`CYCLE` error for both examples.                                                                                                                                                                                                      | Dependencies are collected during evaluation.<br><br>`0` for both examples.                                                    | Same as Google Sheets.                                                                                                         |
+| Named expressions                                  | SALARY:=$A$10 COST:=10*$B$5+100<br>PROFIT:=SALARY-COST<br>A1:=SALARY-COST | Only absolute addresses are allowed<br>(e.g. SALARY:= $A$10).<br><br>Named expressions can be global or scoped to one sheet only.<br><br>They can contain other named expressions.                                                                                                                                                                           | Named expressions are not available.<br><br>Named ranges can be used to create aliases for addresses and ranges.               | Named ranges and scoped named expressions are available.                                                                       |
+| Named expression names                             | ProductPrice1:=42                                                         | A name must be distinctive from a cell reference (case-insensitive), so `ProductPrice1` is invalid.                                                                                                                                                                                                                                                          | A name that is a valid cell reference is allowed if the column address is at least 4-letter long, so `ProductPrice1` is valid. | A name that is a valid cell reference is allowed if the column address is at least 4-letter long, so `ProductPrice1` is valid. |
```

Comment:
Maybe you could add here a link to full description of named expressions accepted formats that is present in: `docs/guide/named-expressions.md`, what do you think?

Reply:
Added